pkg/injector/pod_patch.go

```diff
@@ -408,7 +409,7 @@ func getSidecarContainer(annotations map[string]string, id, daprSidecarImage, na
 		ReadinessProbe: &corev1.Probe{
```

Comment:
Open question to reviewers: do you want probe handler to be extracted into single function ? And then have test for this function?

Reply:
@piotrmsc good suggestion. if you can do, please split it to the function and add some test.

Reply:
@youngbupark please take a look ;-) if you want more things to be split/extracted ping me